x-pack/plugins/translations/translations/fr-FR.json

```diff
@@ -3900,25 +3900,6 @@
     "home.sampleData.logsSpec.webTrafficTitle": "[Logs] Trafic Internet",
     "home.sampleData.logsSpecDescription": "Exemple de données, de visualisations et de tableaux de bord pour le monitoring des logs Internet.",
     "home.sampleData.logsSpecTitle": "Exemple de logs Internet",
-    "homePackages.sampleDataSet.installedLabel": "{name} installé",
```

Comment:
@fkanout Similar issue was bothering me in my recent open PRs, where translation files contained changes that shouldn't be part of my PR. It looks like some indentation or tab/space change in these files. In my PRs I kept in the end only the necessary changes related only to rules page for example. I just wanted to make this comment, because I think unnecessary indentation changes will keep appearing in the PRs, until the problem is fixed in its root.

Reply:
Is the only change about ordering the keys alphabetically?

Reply:
after merging the `main` branch into mine, the issue is solved, I'm seeing only my changes